mypy/errors.py

```diff
@@ -645,7 +645,7 @@ def add_error_info(self, info: ErrorInfo) -> None:
         if ignored_codes and info.code:
             # Something is ignored on the line, but not this error, so maybe the error
             # code is incorrect.
-            msg = f'Error code "{info.code.code}" not covered by "type: ignore" comment'
+            msg = f'Error code "{info.code.code}" not covered by "type: ignore" comment ignoring [{", ".join(ignored_codes)}]'
```

Comment:
If theres no codes IMO it should say `ignoring everything`. mypy-primer says it says `ignoring ` (extra space is intentional).

Reply:
I agree that `ignoring everything` would be preferable to `ignoring ` in this case, but how do you even do that? If it ignores everything how would something not be covered? Also, wouldn't it be `ignoring []`?

Reply:
Oh it is, I misread mypy primer. And this branch doesn't happen for type ignores without a [..]. So uh never mind.